### PR TITLE
Fix footer inconsistencies

### PR DESCRIPTION
**Pull request by GustavoMiguelValencia**

### CODE REVIEW

File: geeksurvey/templates/home.html [hunk 1]
```diff
@@ -4,149 +4,147 @@
 <html>
   <head>
     {% include 'include_bootstrap.html' %}
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="description" content="" />
-    <meta name="author" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <meta name="description" content=""/>
+    <meta name="author" content=""/>
```
Comment (timgiroux) [inline geeksurvey/templates/home.html:9]: didnt see this before but we should probably write something for description and author. in this and base.html

File: geeksurvey/templates/home.html [hunk 2]
```diff
@@ -4,149 +4,147 @@
...
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </section>
-  {% endif %}
-  {% include 'footer.html' %}
+  <body class="d-flex flex-column min-vh-100">
```
Comment (timgiroux) [inline geeksurvey/templates/home.html:16]: :+1:

File: geeksurvey/templates/footer.html [hunk 3]
```diff
@@ -4,15 +4,8 @@
...
-    <a class="m-0 text-white-50 text-decoration-none" href="{% url 'working' %}"> Help </a>
-  </p>
-  <p>
-    <a class="m-0 text-white-50 text-decoration-none" href="https://ceias.nau.edu/capstone/projects/CS/2022/SuperGeeks/"> About </a>
-  </p>
-  <p>
-    <a class="m-0 text-white-50 text-decoration-none" href="https://github.com/NAU-SuperGeeks/geeksurvey"> Source Code </a>
-  </p>
-  <p class="m-0 " style="color:#666;"> Copyright &copy; GeekSurvey 2022 </p>
+    <p class="m-0"> <a class="text-white-50 text-decoration-none" href="{% url 'working' %}"> Help </a></p>
```
Comment (timgiroux) [inline geeksurvey/templates/footer.html:7]: :+1: didnt realize m-0 on these p tags was keeping the padding low. Looks better.

**Review by timgiroux — CHANGES_REQUESTED** [review]
Not *exactly* related to this PR but I think its worthwhile to add a proper description in "meta description" and author in 
"meta author"
see comments

File: geeksurvey/templates/footer.html (hunk 3 above)
Comment (GustavoMiguelValencia) [reply geeksurvey/templates/footer.html:7]: Yup! It's a Bootstrap margin class :).

https://getbootstrap.com/docs/4.0/utilities/spacing/

File: geeksurvey/templates/home.html (hunk 1 above)
Comment (GustavoMiguelValencia) [reply geeksurvey/templates/home.html:9]: Given that multiple people have worked on the files, how should I fill the author? Also I'll working on the descriptions

Comment (GustavoMiguelValencia) [reply geeksurvey/templates/home.html:9]: Actually given that different pages contain different content and have different authors, wouldn't it be best to edit the author and description meta tags in each individual page and not in the base.html file?

Comment (timgiroux) [reply geeksurvey/templates/home.html:9]: hmm yea good point. We can either be cryptic and put the author as "SuperGeeks" for everything or just nuke the author tag.
I think the description tag should stay and get a nice sentence for SEO (it shows up on Google search results and helps get the site indexed)

Comment (GustavoMiguelValencia) [reply geeksurvey/templates/home.html:9]: I'll go ahead and use "SuperGeeks" as the author as the project may be extended upon in the future by someone other than us. I'll go ahead and add a customized description for pages likely to show up in search engine results (home page, log in/sign up pages, etc.).

**Review by timgiroux — APPROVED** [review]

**Comment by timgiroux** [comment]
Nice additions.

**Pull request merged**